### PR TITLE
Hub updates

### PR DESCRIPTION
**Pull request by martypowell**
- Changed hub to use services hub
- Updated html to style buttons

### CODE REVIEW

File: js/page-specific/phone-directory/phone-directory.html [hunk 1]
```diff
@@ -133,62 +125,83 @@
...
+              Submit
+            </button>
+            <button
               id="btnClear"
               name="btnClear"
+              class="seButton contact-btn clear-btn"
               type="button"
-              value="Clear"
-              tabindex="5"
-              class="btnWidths"
```
Comment (martypowell) [inline js/page-specific/phone-directory/phone-directory.html:190]: @srutherfordbcg the important part here is the buttons changing from input elements to buttons elements.

File: js/page-specific/phone-directory/phone-directory.js [hunk 2]
```diff
@@ -2,107 +2,102 @@ var jQuery = $.noConflict(true);
...
+    if (validateForm()) {
+      jQuery("#output").text("");
+      dataString =
+        jQuery(this).closest("form").serialize() +
+        "&formId=" +
+        jQuery(this).closest("form").attr("id");
+
+      jQuery.ajax({
+        url:
+          "https://testservices.baltimorecountymd.gov/api/hub/phoneDirectory/ProcessPhoneDirSearchForm",
```
Comment (martypowell) [inline js/page-specific/phone-directory/phone-directory.js:56]: @srutherfordbcg this is the important change here, the rest is just a reformat.

Comment (srutherfordbcg) [reply js/page-specific/phone-directory/phone-directory.js:56]: Looks pretty straight forward to me.

File: js/page-specific/phone-directory/phone-directory.html (hunk 1 above)
Comment (srutherfordbcg) [reply js/page-specific/phone-directory/phone-directory.html:190]: Looks good - It does seem redundant to have a button object with a type "button" - I wonder if that is still necessary.

**Review by srutherfordbcg — APPROVED** [review]
Looks pretty straightforward. Thanks for the update.

File: js/page-specific/phone-directory/phone-directory.html (hunk 1 above)
Comment (martypowell) [reply js/page-specific/phone-directory/phone-directory.html:190]: @srutherfordbcg some light reading on the topic, https://dev.to/clairecodes/why-its-important-to-give-your-html-button-a-type-58k9

**Pull request merged**